crates/bevy_color/src/testing.rs

```diff
@@ -13,3 +13,21 @@ macro_rules! assert_approx_eq {
 
 #[cfg(test)]
 pub(crate) use assert_approx_eq;
+
+#[cfg(test)]
+macro_rules! call_for_every_color {
```

Comment:
I understand what you're going for with this macro, but I think leave it out of this PR just to keep the focus narrow. Since you're only using it once anyway, might as well move the contents into the test itself:

```rust
#[test]
fn test_gray() {
    verify_gray::<crate::Hsla>();
    verify_gray::<crate::Hsva>();
    // ...
}
```

My concern is we don't actually include every colour in this macro (`Color` in particular) so the name is misleading, and it's fairly rigid in utility (can only inject a single type parameter and no arguments). Plus, macros are still less desirable than just plain Rust code where possible as a general consensus.

Reply:
Done, I can see the reasons for not using them. If it comes to it, that could be refactored in if it had more utility later on!